erpnext/patches/v13_0/set_auto_create_invoice_on_delivery_note.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
+def execute():
+	frappe.reload_doc("stock", "doctype", "accounts_settings")
+
+	create_invoice_on_submit = frappe.db.get_single_value("Accounts Settings", "auto_create_invoice_on_delivery_note_submit")
```

Comment:
Check if the `auto_create_invoice_on_delivery_note_submit` column exists before doing this call. Refer to `frappe.db.has_column` or something like that